src/CheckoutSdk/Payments/Action.cs

```diff
@@ -0,0 +1,20 @@
+using Checkout.Common;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Payments
+{
+    public class Action : Resource
```

Comment:
Please rename to `PaymentAction`

Reply:
Will fix in [this PR](https://github.com/CKOTech/checkout-sdk-net/pull/9) to minimize conflicts.